orsopy/fileio/base.py

```diff
@@ -468,6 +485,42 @@ class Column(Header):
     yaml_representer = Header.yaml_representer_compact
 
 
+@orsodataclass
+class ErrorColumn(Header):
+    """
+    Information about a data column.
+    """
+
+    error_of: str
+    error_type: Optional[Literal["uncertainty", "resolution"]] = None
+    distribution: Optional[
+        Tuple[Literal["gaussian", "triangular", "uniform", "lorentzian"], Literal["sigma", "FWHM"]]
+    ] = None
+
+    yaml_representer = Header.yaml_representer_compact
+
+    def to_sigma(self):
+        """
+        Returns the multiplicative factor needed to convert a FWHM to sigma.
```

Comment:
Is there a reference for these conversions? In particular the triangular and uniform ones?

Reply:
Well, I got them out of my memory and checked with Wikipedia, but I can see if I find a text book to reference here.

Reply:
I hadn't ever thought about the conversations for triangular and uniform is all. Happy with your memory!

Reply:
I added some comments and reference now.